isis/src/base/objs/IrregularBodyCameraGroundMap/unitTest.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+#include "Preference.h"
+
+using namespace std;
+int main() {
+  Isis::Preference::Preferences(true);
+  cout << "This class will be tested by the applications and the individual Camera models." << endl;
+}
```

Comment:
Why even include it then? We really rather have more Makefile based tests.

Reply:
Removed.